### ISSUE TITLE
Is it possible to set variable Lr per epoch 

### ISSUE THREAD

**Issue by gauravbrills** [abhishekkrthakur/tez#33]
@abhishekkrthakur  Was finding this framework great and easy to use . But as fairly new to it was thinking if there is a way to pass variable Lr for training say for every epoch as an example. 

Also is there a way to say continue training from a particular epoch if say the local system crashed or got disturbed during the training process.

**Comment by gauravbrills** [comment]
Ok I think I got it the overridden Model has lot of hooks and **,self.current_epoch** to handle this .. Will check to find out how to continue training from an epoch though

**Comment by abhishekkrthakur** [comment]
Why dont you use StepLR scheduler from pytorch:  https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html

**Comment by gauravbrills** [comment]
> Why dont you use StepLR scheduler from pytorch: https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html

Great thanks @abhishekkrthakur  yes that will work as well
